{{cookiecutter.project_slug}}/dataset.py

```diff
@@ -1,14 +1,40 @@
-from lightning import LightningDataModule
+import lightning as pl
```

Comment:
Import lightning as L, not pl; it confuses it with the older version `PyTorch Lightning`